app/client/components/productPage.tsx

```diff
@@ -173,9 +175,185 @@ const getPaymentPart = (
-) => {
+const getProductDetailRenderer = (
+  originalProductType: ProductType,
+  productPageProperties: ProductPageProperties,
+  productDetailListLength: number
+) => (productDetail: ProductDetail, listIndex: number) => {
+  const productType = originalProductType.mapSoCalledToSpecific
+    ? originalProductType.mapSoCalledToSpecific(productDetail)
+    : originalProductType;
+  const alternateManagementCtaLabel =
+    productType.alternateManagementCtaLabel &&
+    productType.alternateManagementCtaLabel(productDetail);
+  return (
+    <div
+      key={productDetail.subscription.subscriberId}
+      css={{
+        background:
+          productDetailListLength > 1 && (listIndex + 1) % 2 !== 0
+            ? palette.neutral["7"]
+            : undefined,
+        padding: "5px 0 20px"
+      }}
+    >
+      {productDetail.subscription.cancelledAt ? (
+        getCancellationSummary(productType)(productDetail.subscription)
+      ) : (
+        <>
+          {productDetailListLength > 1 ? (
+            <PageContainer noVerticalMargin>
+              {productType.productPage === productPageProperties ? (
+                <h2>
+                  {startCase(productType.friendlyName.toLowerCase())}{" "}
+                  {listIndex + 1}
+                </h2>
+              ) : (
+                <h2>{productDetail.tier}</h2>
+              )}
+            </PageContainer>
+          ) : (
+            undefined
+          )}
+          {productDetail.alertText ? (
+            <div
+              css={{
+                backgroundColor: palette.red.dark,
+                color: palette.white,
+                padding: "10px 15px 15px",
+                margin: `30px ${productDetailListLength > 1 ? "15px" : "0"}`
+              }}
+            >
+              <PageContainer noVerticalMargin>
+                <h2 css={{ fontWeight: "bold", margin: "0" }}>
+                  Action required
+                </h2>
+                <p
+                  id="mma-alert-text"
+                  css={{
+                    br: {
+                      display: "none",
+                      [minWidth.tablet]: {
+                        display: "inline"
+                      }
+                    }
+                  }}
+                >
+                  {alertTextWithoutCTA(productDetail)}
+                  <br />
+                  Please check that the payment details shown are up to date.
+                </p>
+                <LinkButton
+                  text="Update payment details"
+                  to={"/payment/" + productType.urlPart}
+                  height="42px"
+                  fontWeight="bold"
+                  state={productDetail}
+                  primary
+                  right
+                />
+              </PageContainer>
+            </div>
+          ) : (
+            undefined
+          )}
+          <PageContainer>
+            {productPageProperties.showSubscriptionId ? (
```

Comment:
I'd be interested to know why we still need to expose the subscription ID here?

Reply:
Carried over from both /manage and the Digital Pack tab on profile. I'm all for removing this in the long run, but it should be a separate change.
As it stands, at the very least, users need to be shown this so they can use the manual online cancellation form (only membership and contributions have the 3 step cancellation flow in manage)

Reply:
>  I'm all for removing this in the long run, but it should be a separate change.

Definitely - thanks for explaining